extension/org.winepak.Platform.Extension.dxvk-61.yml

```diff
@@ -0,0 +1,95 @@
+build-extension: true
+
+id: org.winepak.Platform.Extension.dxvk-61
+branch: 3.0
+
+runtime: org.winepak.Platform
+runtime-version: 3.0
+sdk: org.winepak.Sdk
+
+separate-locales: false
+appstream-compose: false
+
+modules:
+  - name: dxvk
+    buildsystem: simple
+    build-commands:
+      - install -d ${FLATPAK_DEST}/bin/
+      - install -d ${FLATPAK_DEST}/dlls/x32/
+      - install -d ${FLATPAK_DEST}/dlls/x64/
+      - install -D x32/d3d11.dll ${FLATPAK_DEST}/dlls/x32/
+      - install -D x64/d3d11.dll ${FLATPAK_DEST}/dlls/x64/
+      - install -D x32/dxgi.dll ${FLATPAK_DEST}/dlls/x32/
+      - install -D x64/dxgi.dll ${FLATPAK_DEST}/dlls/x64/
+    sources:
+      - type: archive
+        url: https://github.com/doitsujin/dxvk/releases/download/v0.61/dxvk-0.61.tar.gz
+        sha256: d04388f026dc0d8b276b08f7db74fb3556cbbc8f762401eb5ef52629ee39ded1
+
+  - name: scripts
+    buildsystem: simple
+    build-commands:
+      - install dxvk-install* ${FLATPAK_DEST}/bin
+    sources:
+      - type: script
+        only-arches:
+          - x86_64
+        dest-filename: dxvk-install64
+        commands:
+          - if [ -z "$WINEPREFIX" ] ; then
+          - '    echo "No wine prefix set or is empty, abort."'
+          - '    exit 1'
+          - fi
+          -
+          - EXTENSION_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )/../" && pwd )"
+          -
+          - mkdir -p "${WINEPREFIX}/dosdevices/c:/windows/system32/"
```

Comment:
I looked more into dxvk, the following code won't actually install & enable it. Instead we need to use the `setup_dxvk.sh`.

```
dlls/x64/setup_dxvk.sh -y -q # For dxvk-install64
dlls/x32/setup_dxvk.sh -y -q # For dxvk-install64-wow64 & dxvk-install32
```

Need to re-include the `setup_dxvk.sh` script that was stripped out above.

Reply:
I tried to use these scripts while experimenting with the Fortnite installer with no luck. I'll look more into it tomorrow.